### DISCUSSION TITLE
"Unable to authenticate connection"

### DISCUSSION THREAD

**Discussion by le4onardo** [community/community#135381] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi all, today I tried to open github codespaces but it shows this message:
![Screenshot 2024-08-10 at 21.43.24.png](https://github.zendesk.com/attachments/token/4CX5ahRUis2EhqXKSWBllgyRQ/?name=Screenshot+2024-08-10+at+21.43.24.png)

I tried:
- The retry button, clearing local storage & cookies in both my `github.com` and the `codespace.github.dev` domain.
- Solutions mentioned in this previous, kind of similar, post: 
https://github.com/orgs/community/discussions/18905
- Apparently the docs on the subject: 
https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-your-connection-to-github-codespaces#we-were-unable-to-authenticate-your-connection

But nothing works. Any help will be appreciated :)

**Comment by srijon57** [comment, marked as answer]
Hi there my friend 😊,

It sounds like you've tried a number of troubleshooting steps to resolve the issue with GitHub Codespaces, but you're still encountering problems. Let's try a few more things:

Check Network Connectivity: Ensure that there are no network restrictions or firewall settings that might be blocking the connection to GitHub Codespaces. Sometimes, a VPN or certain network configurations can cause issues.

Try a Different Browser or Incognito Mode: Sometimes, browser extensions or settings can interfere with Codespaces. Try opening GitHub Codespaces in a different browser or in incognito/private mode to see if that resolves the issue.

Disable Any Proxy Settings: If you're using a proxy, try disabling it temporarily to see if it resolves the connection issue.

Clear DNS Cache: If you're comfortable with it, clearing your DNS cache might help. You can do this by running the following command in your terminal (for Windows, use Command Prompt):

On Windows:
ipconfig /flushdns

On macOS and Linux:
sudo dscacheutil -flushcache; sudo killall -HUP mDNSResponder

After clearing the DNS cache, restart your browser and try accessing Codespaces again.
Check GitHub Status: Occasionally, the issue might be related to GitHub's services being down or experiencing difficulties. You can check (https://www.githubstatus.com/) to see if there are any ongoing issues.

Reach Out to GitHub Support: If none of the above steps work, I would recommend reaching out to (https://support.github.com/contact) with detailed information about the issue. They may be able to provide further assistance.

Let me know if any of these steps help.

**Reply by le4onardo** [reply]
> Sometimes, browser extensions or settings can interfere with Codespaces.

I forgot I had an extension called "CORS Unblock". Once disabled my codespace loaded as expected.
Good catch, thank you !
